Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -390,7 +390,7 @@ Layer* ImageEditor::layer_at_editor_position(Gfx::IntPoint const& editor_positio
 
 void ImageEditor::clamped_scale(float scale_delta)
 {
-    m_scale += scale_delta;
+    m_scale *= static_cast<float>(exp2(scale_delta));
```

Comment:
if it is using `AK/Math` it return float on it own
otherwise `expf`

Reply:
Changed to `AK::exp2`.